src/app/page.tsx

```diff
@@ -1,9 +1,21 @@
 "use client"
-import { Box } from "@mui/material";
+import { Box } from "@mui/material"
+import { useRouter } from "next/navigation"
+import { useEffect, useState } from "react"
 
 export default function Home() {
-  return (
-    <Box>
-    </Box>
-  );
+  const router = useRouter()
+  const [isMounted, setIsMounted] = useState<Boolean>(false)
+
+  useEffect(() => {
+    setIsMounted(true)
+  }, [])
+
+  useEffect(() => {
+    if (isMounted) {
+      router.push("/Nivelamento")
+    }
+  }, [isMounted, router])
+
+  return <Box></Box>
```

Comment:
Essas alteracoes foram pegas do card 74, quando ele for aprovado lembrem-se de puxarem as alterações da main